package/MDAnalysis/analysis/backends.py

```diff
@@ -0,0 +1,333 @@
+"""Analysis backends --- :mod:`MDAnalysis.analysis.backends`
+============================================================
+
+.. versionadded:: 2.8.0
+
+
+The :mod:`backends` module provides :class:`BackendBase` base class to
+implement custom backends for
```

Comment:
```suggestion
implement custom execution backends for
```
Or something like that, otherwise it's a bit hard to understand what backend is from these docs.